src/go/rpk/go.mod

```diff
@@ -1,5 +1,7 @@
 module github.com/vectorizedio/redpanda/src/go/rpk
 
+go 1.16
```

Comment:
Based on you comment that go 1.17 have new method in `reflect` package why not bump strait to 1.17? Are there some problems that I'm not aware of?

Reply:
:thinking: imo there's no concern. I'll try bumping this (and changing the `out` code back to using `IsExported`) to see if any build failures crop up.

Reply:
Yeah this complicates things. The k8s subdirector in particular gets particularly screwed up with 1.17 and how mod changed. This can be done in a followup PR.